webpack.config.js

```diff
@@ -135,15 +135,15 @@ module.exports = {
       },
       { enforce: "pre", test: /\.js$/, loader: "source-map-loader" },
       {
-        test: /\.(jpg|png)$/,
+        test: /\.(jpg|png|svg)$/,
```

Comment:
why do you change this? i am pretty sure that it was done on purpose (i changed it to make it work)

Reply:
please take a look at other usages of SVGs in react app to make sure that they still work